build/CoreText/lib/CoreTextLib.vcxproj

```diff
@@ -37,7 +37,7 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreText\DWriteWrapper.mm" />
   </ItemGroup>
   <ItemGroup>
-    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreText\DWriteWrapper.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\include\CoreText\DWriteWrapper.h" />
```

Comment:
never mind, it is still private.

---
In reply to: [78118093](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78118093) [](ancestors = 78118093)